dotCMS/src/main/java/com/dotcms/rest/elasticsearch/ESContentResourcePortlet.java

```diff
@@ -117,21 +113,15 @@ public Response searchRaw(@Context HttpServletRequest request) {
 
 		HttpSession session = request.getSession();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "session". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "session" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)